lib/tasks/import.rake

```diff
@@ -70,38 +66,41 @@ namespace :import do
       o.save(validate: false)
       organisation_count += 1
     end
-
     logger.info "Imported #{organisation_count} organisations...\r"
-    logger.info "Processing #{relationships.size} child organisations"
 
+    logger.info "Processing #{relationships.size} child organisations"
     relationships.each do |child, parent|
       o = Organisation.find_by(name: child)
       o.parent = Organisation.find_by(name: parent)
       o.save!(validate: false)
       child_organisation_count += 1
     end
-
     logger.info "Assigned #{child_organisation_count} organisations...\r"
-    logger.info "Done"
+
+    logger.info "Import complete"
   end
 
   desc "Import datasets from a data.gov.uk dump"
-  task :datasets, [:filename] => :environment do |_, args|
+  task :datasets => :environment do |_, args|
     Link.skip_callback(:save, :before, :set_date)
 
     # Maps the organisation UUIDs to the organisation IDs
+    logger = Logger.new(STDOUT)
     orgs_cache = Organisation.all.pluck(:uuid, :id).to_h
     theme_cache = Theme.all.pluck(:title, :id).to_h
     counter = 0
+    host = 'https://data.gov.uk/'
+    path = 'data/dumps/data.gov.uk-ckan-meta-data-latest.v2.jsonl.zip'
+    url = URI::join(host, path).to_s
+    file = download_data('latest_legacy_datasets', url, logger)
```

Comment:
Data will be downloaded automatically, rather than making you do it yourself :-)